javascript-forms/form-values.js

```diff
@@ -0,0 +1,16 @@
+var $container = document.querySelector('#contact-form');
+$container.addEventListener('submit', handleSubmit);
+
+var $name = $container.elements.name.value;
+var $email = $container.elements.email.value;
+var $message = $container.elements.message.value;
+var $formObject = {};
```

Comment:
This object should be declared within the handleSubmit function because you want to have a different object created each time the form is submitted.